build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -136,9 +148,10 @@
     torch_models.MODEL_CLIP_TEXT_SEQLEN64_FP32_TORCH,
     torch_models.MODEL_UNET_2D_FP32_TORCH,
     torch_models.EFFICIENTNET_B7_FP32_TORCH,
-]
+] + BERT_LARGE_FP16_TORCH_BATCHES
```

Comment:
Do we need to benchmark all batch sizes in the default benchmark suite? Maybe just one batch size and the rest can go into `CUDA_MODELS_LONG`?

Reply:
+1 on this. However you might run into an issue where the same model cannot be used in `CUDA_MODELS` and `CUDA_MODELS_LONG`. I believe @pzread is working on a fix.

Reply:
SGTM, I'll move all but one of the batch sizes to CUDA_LONG.

Reply:
Yes, support benchmark in multiple presets in tracked on #13683